integration/tests/carvel/04-default-deployment.spec.js

```diff
@@ -24,7 +24,8 @@ test("Deploys package with default values in main cluster", async ({ page }) =>
 
   // Deploy package
   await page.waitForSelector('select[name="package-versions"]');
-  await page.selectOption('select[name="package-versions"]', "2.0.0");
+  const versionSelector = await page.$('select[name="package-versions"]')
+  await versionSelector?.selectOption("2.0.0")
```

Comment:
I think (not sure though) you can use the page.locator (which should auto-wait) and then use the `selectOption`, like

````js
  await page.locator("select[name="package-versions"]").selectOption("2.0.0");
````

But, if your fix is proven to work, +1. Thanks for fixing it, it was generating some headaches recently!

Reply:
The underlying problem is not that the `select` control is not found, but that the option selected with `selectOption` was not being applied. Analyzing the failing tests dump shows that Kubeapps backend was receiving version `3.0.0-rc.1` (default one).

Let's see if this consistently fixes the problem, otherwise we'll have to try other approaches.